docs/api/api_blueprint.apib

```diff
@@ -5937,6 +5937,202 @@ Delete a single image size.
           }
         }
 
+# Group Ticket Fees
+
+Ticket Fees basically stores the fee on a ticket
+
+| Parameter | Description | Type | Required |
+|:----------|-------------|------|----------|
+| `currency`  | Currency of the ticket | string | - |
+| `service_fee` | Service fee the ticket  | float | - |
+| `maximum_fee` | Maximum fee the ticket  | float | - |
+
+## Image Size Collection [/v1/ticket-fees{?page%5bsize%5d,page%5bnumber%5d,sort,filter}]
```

Comment:
Correct Image Size to Ticket fees.
Please place this ticket fees section after tickets to keep the arrangement proper.
And as per [#3849(comment)](https://github.com/fossasia/open-event-orga-server/pull/3849#issuecomment-310365677), start adding the test to hook_main.py everytime you add in a new api in api doc